core/views.py

```diff
@@ -14,6 +17,19 @@ def get_page_size(self, request):
             return self.max_page_size
         return super().get_page_size(request)
 
+# Permit to send feedback emails from users
+@api_view(['POST'])
+def feedback(request, format=None):
+    data = request.data
+    if len(data['message']) < 2:
+        return HttpResponse(status=500)
+
+    subject = "[WOOLLY][FeedBack] - " + data['reason'] + " - " + data['sender']['name']
+    message = 'Utilisateur : ' + data['sender']['name'] + '\n' + 'ID : ' + data['sender']['id'] + '\n' +  'Email : ' + data['sender']['email'] + '\n' + 'Date : ' + datetime.now().strftime("%d/%m/%Y, %H:%M:%S") + '\n' + 'Raison : ' + data['reason'] + '\n \n' + 'Message :' + '\n' + data['message']
+    message = message + '\n\nCe message a été généré automatiquement, merci de ne pas y répondre'
```

Comment:
```suggestion
    subject = f"[WOOLLY][FeedBack] - {data['reason']} - {data['sender']['name']}"
    message = "\n".join([
        f"Utilisateur: {data['sender']['name']}",
        f"Identifiant: {data['sender']['id']}",
        f"Email: {data['sender']['email']}",
        f"Date: {datetime.now().isoformat(sep=' ', timespec='seconds')}"
        f"Raison: {data['reason']}",
        "",
        {data['message']},
        "",
        "Ce message a été généré automatiquement, merci de ne pas y répondre",
    ])
```
You should also check that all the data attributes are present, maybe create a Model and a Serializer ?